packages/stack/src/views/Stack/Card.tsx

```diff
@@ -336,6 +336,10 @@ export class Card extends React.Component<Props> {
         onGestureEnd?.();
         break;
       }
+      case GestureState.FAILED: {
+        this.handleEndInteraction();
```

Comment:
I'm not sure if this is enough. Isn't `this.isSwiping.setValue(FALSE);` needed as well?

Reply:
Hmm, maybe. Thank you for review! I'll check it

Reply:
done